test_queue_job/tests/test_job_auto_delay.py

```diff
@@ -32,11 +32,16 @@ def test_auto_delay_inside_job(self):
 
     def test_auto_delay_force_sync(self):
         """method forced to run synchronously"""
-        result = (
-            self.env["test.queue.job"]
-            .with_context(_job_force_sync=True)
-            .delay_me(1, kwarg=2)
+        with self.assertLogs(level="WARNING") as log_catcher:
```

Comment:
this test needs an update no?

Reply:
Nope, corresponding to true warnings: https://github.com/OCA/queue/pull/679/files#diff-6e33951ef77e0428f3edcd938952cf20d30ca84e703406c08685f3d70d68da86R28